dotnet-test: add page (#12815)
tldr-pages/tldr: pages/common/dotnet-test.md

# dotnet test

> > Execute tests for a .NET application.

Reviewer: ```suggestion
> Execute tests for a .NET application.
> Note: View <https://learn.microsoft.com/en-us/dotnet/core/testing/selective-unit-tests#syntax> for supported filter expressions.
```

Author: Okay, I believe on the third line after the `> More information (...)` line would be good.

> More information: <https://learn.microsoft.com/dotnet/core/tools/dotnet-test>.

> - Execute tests for a .NET project / solution in the current directory:

Reviewer: ```suggestion
- Execute tests for a .NET project/solution in the current directory:
```
We generally don't add spaces between slashes in descriptions (so we could do the same here to stay consistent with other pages having similar syntax).

Author: Thank you for the review, I wasn't sure about this, so it's now clear :).

`dotnet test`

> - Execute tests for a .NET project / solution in a specific location:

Reviewer: ```suggestion
- Execute tests for a .NET project/solution in a specific location:
```

`dotnet test {{path/to/project_or_solution}}`

> - Execute tests whose name contains `TestMethod1`:
> 
> `dotnet test --filter Name~TestMethod1`

Reviewer: ```suggestion
- Execute tests with a specific filter:

`dotnet test --filter {{Name~TestMethod1}}`
```

Maybe we could generalize this example and also convert the filter to a placeholder.

Author: Could I even put a link to the documentation for those filters? It would be this one then: https://learn.microsoft.com/en-us/dotnet/core/testing/selective-unit-tests?pivots=xunit
